### ISSUE TITLE
[Feature Request] Low memory set alert.

### ISSUE THREAD

**Issue by Slayer5934** [DimensionalDevelopment/VanillaFix#171]
As a modpack creator it would be useful if you can let the user know they have too little memory allocated to minecraft.

Features:
- Comes disabled, but can be enabled in config.
- If memory amount is less-than amount config.
- Alert pops up before starting game (in loading) and/or in the event of a crash.

Thank you for VanillaFix, it makes life easier for pack creators and the players.

**Comment by LokeYourLord** [comment]
Yeah it would be great if VanillaFix could merge with [MemoryTester](https://minecraft.curseforge.com/projects/memorytester), as MemoryTester only checks for the **_specific_** amount, not if it is equal or above the specified amount, meaning that is constantly gives a warning if the user even has more allocated than recommended.

**Comment by xJon** [comment]
Use [Modpack Configuration Checker](https://www.curseforge.com/minecraft/mc-mods/config-checker).
